src/index.js

```diff
@@ -1,14 +1,65 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+let newTask;
+const arrowBtn = document.getElementById('arrow');
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+class Todolist {
+  constructor(description, completed, index) {
```

Comment:
- [ ] You have not implemented the method to update the to-do description. Please add this method :+1:

Reply:
not fulfilled 🚫